packages/create-redwood-app/package.json

```diff
@@ -19,8 +19,9 @@
     "build:watch": "nodemon --watch src --ignore dist,template --exec \"yarn build\"",
     "prepublishOnly": "NODE_ENV=production yarn build",
     "set-up-test-project": "node ./scripts/setUpTestProject.js",
-    "test": "node --experimental-vm-modules $(yarn bin jest) templates",
-    "test:e2e": "node --experimental-vm-modules $(yarn bin jest) e2e",
+    "test": "vitest run templates",
+    "test:watch": "vitest watch templates",
```

Comment:
Did you use this one or were just copying from other packages? Not sure if the templates test is a great one to watch since it runs against dist. I also just don't use the watch scripts though

```suggestion
```

Reply:
Copied from the other packages. I assumed we might want to have it supported everywhere if we support it anywhere but you're right it doesn't need to come in here.